src/PrestaShopBundle/Twig/HookExtension.php

```diff
@@ -62,7 +62,7 @@ class HookExtension extends AbstractExtension
     public function __construct(
         HookDispatcherInterface $hookDispatcher,
         ModuleDataProvider $moduleDataProvider,
-        ModuleRepository $moduleRepository = null
+        ModuleRepositoryInterface $moduleRepository = null
```

Comment:
```suggestion
        ?ModuleRepositoryInterface $moduleRepository = null
```

Reply:
Perhaps because if there is no employee in context, then there is no HTML to render ?
Looks like pure luck :))